content/en/metrics/custom_metrics.md

```diff
@@ -77,7 +77,6 @@ Set metric units through [Metrics Summary][12] or set custom metric units with t
     {{< nextlink href="/api/v1/metrics/#submit-metrics" >}}Datadog's HTTP API{{< /nextlink >}}
     {{< nextlink href="/logs/log_configuration/logs_to_metrics/#generate-a-log-based-metric" >}}Generate Log-based metrics{{< /nextlink >}}
     {{< nextlink href="/tracing/generate_metrics/" >}}Generate APM span-based metrics{{< /nextlink >}}
-    {{< nextlink href="/continuous_integration/guides/add_custom_metrics/" >}}Generate CI Visibility test-based metrics{{< /nextlink >}}
```

Comment:
I've removed this because these are not actually metrics but numerical tags (measures)